packages/ds/src/components/select/select.stories.mdx

```diff
@@ -0,0 +1,173 @@
+        <option>Mojito</option>
+      </Select>
+      <Select
+        defaultValue="Margarita"
+        rightIcon={<Icons.Info color="text.primary" aria-hidden />}
+        mb={4}
+      >
+        <option>Pina Colada</option>
+        <option>Margarita</option>
+        <option>Mojito</option>
+      </Select>
+      <Select
+        defaultValue="Margarita"
+        leftIcon={<Icons.Search color="text.primary" aria-hidden />}
+        rightIcon={<Icons.Info color="text.primary" aria-hidden />}
+        mb={4}
+      >
+        <option>Pina Colada</option>
+        <option>Margarita</option>
+        <option>Mojito</option>
+      </Select>
+    </Box>
+  </Story>
+</Preview>
+
+#### Disabled
+
+<Preview withToolbar>
+  <Story name="disabled">
+    <Box>
+      <Select
+        disabled
+        defaultValue="Margarita"
+        mb={4}
+      >
+        <option>Pina Colada</option>
+        <option>Margarita</option>
+        <option>Mojito</option>
+      </Select>
+      <Select
+        disabled
+        defaultValue="Margarita"
+        leftIcon={<Icons.Search aria-hidden />}
+        mb={4}
+      >
+        <option>Pina Colada</option>
+        <option>Margarita</option>
+        <option>Mojito</option>
+      </Select>
+    </Box>
+  </Story>
+</Preview>
+
+#### Read Only
+
+<Preview withToolbar>
+  <Story name="read only">
+    <Box>
+      <Select
+        readOnly={true}
+        defaultValue="Margarita"
+        mb={4}
+      >
+        <option>Pina Colada</option>
+        <option>Margarita</option>
+        <option>Mojito</option>
+      </Select>
+      <Select
+        readOnly={true}
+        defaultValue="Margarita"
+        leftIcon={<Icons.Search color="text.primary" aria-hidden />}
+        mb={4}
+      >
+        <option>Pina Colada</option>
+        <option>Margarita</option>
+        <option>Mojito</option>
+      </Select>
+    </Box>
+  </Story>
+</Preview>
+
+#### Error
+
+<Preview withToolbar>
+  <Story name="error">
+    <Box>
+      <Select
+        error={true}
+        defaultValue="Margarita"
+        mb={4}
+      >
+        <option>Pina Colada</option>
+        <option>Margarita</option>
+        <option>Mojito</option>
+      </Select>
+      <Select
+        error
+        defaultValue="Margarita"
+        leftIcon={<Icons.Search color="text.primary" aria-hidden />}
+        mb={4}
+      >
+        <option>Pina Colada</option>
+        <option>Margarita</option>
+        <option>Mojito</option>
+      </Select>
+      <Select
+        error
+        defaultValue="Margarita"
+        rightIcon={<Icons.Info color="text.primary" aria-hidden />}
+        mb={4}
+      >
+        <option>Pina Colada</option>
+        <option>Margarita</option>
+        <option>Mojito</option>
+      </Select>
+      <Select
+        error
+        defaultValue="Margarita"
+        leftIcon={<Icons.Search color="text.primary" aria-hidden />}
+        rightIcon={<Icons.Info color="text.primary" aria-hidden />}
```

Comment:
Should be `<Icons.Error aria-hidden color="ui.error" />` for all the error stories